benchmarks/perf-tool/release-configs/lucene-hnsw/nested/simple/simple-nested-test.yml

```diff
@@ -9,7 +9,7 @@ steps:
     index_name: target_index
   - name: create_index
     index_name: target_index
-    index_spec: release-configs/faiss-hnsw/nested/simple/index.json
+    index_spec: release-configs/lucene-hnsw/nested/simple/index.json
```

Comment:
why we're changing `faiss` to `lucene` here?

Reply:
It was a bug introduced before. This is test spec for lucene.